README.md

```diff
@@ -5,7 +5,7 @@ VOLTTRON base historian framework that provide common functions such as caching,
 
 ## Requirements
 
- - Python >= 3.8
+ - Python >= 3.10
```

Comment:
Add poetry here...I need to do that on the core as well.

Reply:
if core has poetry as requirement, that should do it right?

Reply:
Well then we should have VOLTTRON as a requirement right?  Poetry is really a development requirement...I suppose